packages/contracts-bedrock/test/invariants/CrossDomainMessenger.t.sol

```diff
@@ -57,7 +57,7 @@ contract RelayActor is StdUtils {
 
         // If the message should succeed, supply it `baseGas`. If not, supply it an amount of
         // gas that is too low to complete the call.
-        uint256 gas = doFail ? bound(minGasLimit, 60_000, 80_000) : xdm.baseGas(_message, minGasLimit);
+        uint256 gas = doFail ? bound(minGasLimit, 90_000, 100_000) : xdm.baseGas(_message, minGasLimit);
```

Comment:
Gas required to store a message has increased because `pause` now requires extra calls via the SystemConfig into the SuperchainConfig. Should really be ok in practice because `minGasLimit` was always conservative and overestimated the amount of required gas. We should copy `testFuzz_relayMessage_baseGasSufficient_succeeds` to `L2CrossDomainMessenger.t.sol` to confirm that the current gas formula is fine and no existing messages would be dropped by this change.